scripts/install_godot_linux.rb

```diff
@@ -8,7 +8,7 @@
 require 'tmpdir'
 require 'httparty'
 
-require_relative 'godot_version'
+require_relative 'scripts/godot_version'
```

Comment:
I'm pretty sure this change actually breaks it

Reply:
Because this loads a file relative to this file, which is still in the same folder as the godot_version file